proto/v1/structure/state.proto

```diff
@@ -0,0 +1,455 @@
+syntax = "proto3";
+
+package org.oppia.proto.v1.structure;
+
+import "proto/v1/structure/languages.proto";
+import "proto/v1/structure/objects.proto";
+import "proto/v1/structure/versions.proto";
+
+option java_package = "org.oppia.v1.structure";
+option java_multiple_files = true;
+
+message State {
+  StateStructureVersion structure_version = 1;
+
+  SubtitledHtml content = 2;
+  InteractionInstance interaction = 3;
+  RecordedVoiceovers recorded_voiceovers = 4;
```

Comment:
no `RecordedVoiceovers` message added
`message RecordedVoiceovers{}` - not present

Reply:
Ditto. FWIW the local build works correctly with Bazel (that is, ``bazel build //...`` correctly builds the java proto/proto lite variants).

Reply:
earlier, I didn't find the `message RecordedVoiceovers {}`, it's inside the `language.proto`. I got it. 
Resolving the conversation as I have nothing more to ask here.